Provis.Core/Validation/WorkspaceCreateValidation.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using Provis.Core.DTO.workspaceDTO;
+
+namespace Provis.Core.Validation
+{
+    public class WorkspaceCreateValidation : AbstractValidator<WorkspaceCreateDTO>
+    {
+        public WorkspaceCreateValidation()
+        {
+            RuleFor(workspace => workspace.Name)
+                .NotEmpty()
+                .NotNull()
+                .Length(1, 50)
+                .Matches("^[^£# “”]*$").
+                WithMessage("{PropertyName} must not contain the following characters £ # “” or spaces.");
```

Comment:
.WithMessage should be from new line the character dot should be from new line too